tests/portalbackend/PortalBackend.DBAccess.Tests/CompanyRoleCollectionRolesViewTests.cs

```diff
@@ -44,8 +44,8 @@ public async Task CompanyRoleCollectionRolesView_GetAll_ReturnsExpected()
         var sut = await CreateContext();
 
         // Act
-        var result = await sut.CompanyRoleCollectionRolesView.ToListAsync();
-        result.Should().HaveCount(26);
+        var result = await sut.CompanyRoleCollectionRolesView.ToListAsync().ConfigureAwait(false);
```

Comment:
Please change to, after the upgrade to .NET 8 we upgraded the unit test packages as well, this will result in a warning 

```suggestion
        var result = await sut.CompanyRoleCollectionRolesView.ToListAsync();
```